Source/Scene/Scene.js

```diff
@@ -690,10 +702,17 @@ define([
 
         var camera = new Camera(this);
         this._camera = camera;
-        this._cameraClone = Camera.clone(camera);
         this._screenSpaceCameraController = new ScreenSpaceCameraController(this);
         this._mapMode2D = defaultValue(options.mapMode2D, MapMode2D.INFINITE_SCROLL);
 
+        if (this._logDepthBuffer) {
+            this._camera.frustum.near = 0.1;
```

Comment:
Should this be configurable?

Reply:
It is configurable. The default for the camera is `1.0` and `5e8`. This changes them to better defaults when log depth is enabled. After the scene is created, any value that near and far are set to will be used.